### ISSUE TITLE
yarn bootstrap removed

### ISSUE THREAD

**Issue by globallyunique** [freon4dsl/Freon4dsl#321]
After cloning and following the Read.Me, installing using homebrew, the yarn bootstrap command fails. So the instructions or build be updated?

yarn bootstrap
yarn run v1.22.21
$ lerna bootstrap
ERR! bootstrap The "bootstrap" command was removed by default in v7, and is no longer maintained.
ERR! bootstrap Learn more about this change at https://lerna.js.org/docs/legacy-package-management
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by joswarmer** [comment]
Yes, lerna is not maintained anymore.  It still works on my machine (also Mac), but it definitely needs to be replaced.  Nowadays npm has enough functionality to be used by Freon.

**Comment by joswarmer** [comment]
Has been fixed now
